test_toil.py

```diff
@@ -0,0 +1,2 @@
+import toil
```

Comment:
I assume this file will eventually be a test in ```tests/``` ? Maybe we can remove it for now?

Reply:
Oops, didn't mean to add that file.